lib/vtls/openssl.c

```diff
@@ -2113,6 +2109,7 @@ set_ssl_version_min_max(long *ctx_options, struct connectdata *conn,
 #endif
       /* FALLTHROUGH */
     case CURL_SSLVERSION_TLSv1_0:
+    case CURL_SSLVERSION_TLSv1:
```

Comment:
Adding this case here has the same effect as the existing code at line number 2336. Should that line be removed?

Reply:
Thanks, good catch. It seems wrong to have this duplicated, I'll see what I should cleanup here...